samples/sensor/vl53l1x/README.rst

```diff
@@ -0,0 +1,48 @@
+.. _vl53l1x:
+
+VL53L0X: Time Of Flight sensor
```

Comment:
We are trying to reduce the number of driver-specific sensor sample applications. Can you merge this with the vl53l0x sample?

Reply:
@MaureenHelm Would it be possible to leave my sample as-is for now, then create a new issue to merge the two and assign it to me?

Reply:
No, I don't want to introduce a new sample just to remove it soon after. I would prefer that you remove the sample from this PR and come back with a follow-up PR that updates the sample to support both variants.